examples/locality-load-balancing/envoy-proxy.yaml

```diff
@@ -39,6 +39,8 @@ static_resources:
   - name: backend
     type: STRICT_DNS
     lb_policy: ROUND_ROBIN
+    common_lb_config:
+      locality_weighted_lb_config: {} # Must be set for load_balancing_weight to be used
```

Comment:
i would avoid adding this here - can we add something in the actual example docs here - https://github.com/envoyproxy/envoy/blob/main/docs/root/start/sandboxes/locality_load_balancing.rst (arggh the github ui is useless here)

we probably want to make this a `.. note::` somewhere to draw attention to it

ie

```rst

... note::
    The ``locality_weighted_lb_config`` must be set in ``common_lb_config``
    for the ``load_balancing_weight`` to be used.

```

or similar

Reply:
currently rendered page https://www.envoyproxy.io/docs/envoy/latest/start/sandboxes/locality_load_balancing

probably we could put it at the end of the first section, just after where it mentions the config

Reply:
Fixed, but I messed up the DCO trying to use my full name let me fix that